authentication-service/src/main/kotlin/com/saveourtool/save/authservice/repository/AuthenticationUserRepository.kt

```diff
@@ -13,17 +13,22 @@ class AuthenticationUserRepository(
     private val namedParameterJdbcTemplate: NamedParameterJdbcTemplate,
 ) {
     /**
-     * @param name
+     * @param userNameAndSource
      * @return user or null if no results have been found
      */
-    fun findByName(name: String): User? {
+    fun findByName(userNameAndSource: String): User? {
```

Comment:
suggestion to rename the method or pass two arguments here

Reply:
`com.saveourtool.save.authservice.service.AuthenticationUserDetailsService` is from Spring. 
`AuthenticationUserRepository.kt` is our repository class

Reply:
Changed it